tests/components/airvisual/test_config_flow.py

```diff
@@ -44,6 +43,8 @@ async def test_duplicate_error(hass):
     assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "already_configured"
 
+    node_pro_conf = {CONF_IP_ADDRESS: "192.168.1.100", CONF_PASSWORD: "12345"}
```

Comment:
This is being reused between tests so can be added to the top as a const. Something like `FIXTURE_CONFIG = {CONF_IP_ADDRESS: "192.168.1.100", CONF_PASSWORD: "12345"}`

Reply:
Same comment as above.